.github/workflows/alert-deploy.yml

```diff
@@ -0,0 +1,41 @@
+name: Deploy alerts to NAIS
```

Comment:
Deploy automatisk til prod ved merge til main

Reply:
Tar det i en ny PR, for å kunne teste i preprod først